tools/system_libs.py

```diff
@@ -1446,7 +1447,10 @@ def get_files(self):
     conf_files = files_in_path(
         path_components=['system', 'lib', 'libc', 'musl', 'src', 'conf'],
         filenames=['sysconf.c'])
-    return base_files + time_files + exit_files + conf_files
+    cxxabi_files = files_in_path(
+        path_components=['system', 'lib', 'libcxxabi', 'src'],
+        filenames=['cxa_thread_atexit.cpp', 'cxa_guard.cpp'])
```

Comment:
`cxa_guard.cpp` is also in `libcxxabi`, is that not a problem? I guess not since only one will be linked in, but it still seems odd?

Reply:
Fixed.